### DISCUSSION TITLE
What are the default/base values for the sizes property of the theme spec?

### DISCUSSION THREAD

**Discussion by JakeMetz8** [system-ui/theme-ui#2347] [Q&A] [answered]
If the **sizes** property is not declared in theme what are the default sizes?

I cannot find this anywhere in the documentation nor does it look like the property is set in the base preset [here](https://github.com/system-ui/theme-ui/blob/develop/packages/preset-base/src/index.ts)

**Comment by lachlanjc** [comment, marked as answer]
There aren't any :) The default theme only defines `space` & `fontSizes`—check that out [here](https://github.com/system-ui/theme-ui/blob/develop/packages/css/src/index.ts#L76-L79)—& the idea is you add your own sizes as a user.
